Update landing page

lboeman opened the pull request:
closes #243 
Mostly touched the unauthenticated landing page, which i think is an improvement, and added links and reworded some of the authenticated landing page. Including screenshots in case they're easier to read than the html. 

Authenticated:
![authenticated](https://user-images.githubusercontent.com/21206164/99324032-9e0b1600-2830-11eb-8653-bbef93f9939c.png)

Unauthenticated:
![unauthenticated](https://user-images.githubusercontent.com/21206164/99324040-a19e9d00-2830-11eb-8a92-b57ff3559672.png)

wholmgren commented on sfa_dash/templates/index.html:
```diff
@@ -6,33 +6,58 @@
-the aggregates menu, you will be able to create new aggregates, associate
-existing observations with an aggregate and view aggregate data.</p>
+<p>The <a class="font-weight-bold" href="/reports" >Reports</a>
+link lists Reports available to you and allow you to create new reports.</p>
```
```suggestion
link lists Reports available to you and allows you to create new reports.</p>
```

wholmgren reviewed (commented):
Definitely an improvement for the unauthenticated page.

One concern about the authenticated page: the create options are not available to all users, so the text might cause some confusion.

wholmgren commented on sfa_dash/templates/index.html:
```diff
@@ -6,33 +6,58 @@
+solar power and irradiance forecasts. The Arbiter will allow you to securely
+upload forecast data for your own private evaluation, or to participate in
+competitive forecast trials that compare forecast performance between
+anonymous participantss.</p>
```
```suggestion
anonymous participants.</p>
```

wholmgren reviewed (commented):
here are some quick fixes. I want to come back and tweak another sentence in a bit

wholmgren commented on sfa_dash/templates/index.html:
```diff
@@ -6,33 +6,58 @@
 {% if current_user is none %}
-<p>Welcome to the Solar Forecast Arbiter Dashboard. If you have an account, please
+<p>Welcome to the Solar Forecast Arbiter, a tool for evaluating your
+solar power and irradiance forecasts. The Arbiter will allow you to securely
+upload forecast data for your own private evaluation, or to participate in
+competitive forecast trials that compare forecast performance between
+anonymous participants.</p>
```
can you edit and html-ify the following?

The Arbiter allows you to:

* Securely upload forecast and observation data
* Analyze historical forecast accuracy and track ongoing forecast accuracy
* Optionally share data and analysis reports with other users
* Participate in anonymous, multi-vendor, operational forecast trials

lboeman commented:
Added mention of getting the admin section through the *Account* menu, and added a link to the administration docs on the admin landing page. 
I also reworded the authenticated landing to first mention general features that a new user might have access to and then mention the features accessible with the correct permissions. 
@wholmgren ?
![Screenshot from 2020-11-17 10-20-34](https://user-images.githubusercontent.com/21206164/99424545-0a345b00-28bf-11eb-97e3-ca905afe99d5.png)

wholmgren reviewed (approved):
@alorenzo175 can you give this a quick look for typos and anything else you think is unclear? I don't want to get this one wrong.

alorenzo175 commented on sfa_dash/templates/index.html:
```diff
@@ -6,33 +6,73 @@
+Observations and aggregated observation data. Users with the proper permissions
+can <a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-aggregate">create new Aggregates</a>,
+<a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#add-observations-to-an-aggregate">associate existing Observations with an Aggregate</a>,
+and <a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-forecast">create Forecasts, and Probabilsitic Forecasts</a>
```
```suggestion
or <a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-forecast">create Forecasts and Probabilistic Forecasts</a>
```
